ChatterboxBackend/Chatterbox/Controllers/ChatController.cs

```diff
@@ -0,0 +1,125 @@
+﻿using Chatterbox.Models;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Chatterbox.Controllers
+{
+    [Route("api/[controller]")]
+    public class ChatController:Controller
+    {
+        private readonly ChatterContext _context;
+
+        public ChatController(ChatterContext context)
+        {
+            _context = context;
+        }
+
+        [HttpGet]
+        public IEnumerable<Conversation > GetAll()
+        {
+            return _context.Conversations.ToList();
+        }
+        
+        [HttpPost("newmessage")]
+        public IActionResult Create([FromBody] MessageReceived message)
+        {
+            Console.WriteLine("NEW POST FOR MESSAGE");
+            if (message == null)
+            {
+                Console.WriteLine("ERROR");

```

Comment:
Same as above. Don't print debug messages to the console. Rather use the debugger.